README.md

```diff
@@ -34,17 +34,16 @@ they are last pushed to. This is configurable with the `--graphite.sample-expiry
 
 ## Metric Mapping and Configuration
 
+**Please note there has been a breaking change in configuration after version 0.2.0.  The YAML style config from [statsd_exporter](https://github.com/prometheus/statsd_exporter) is now used.  See conversion instructions below**
+
+### YAML Config
 The graphite_exporter can be configured to translate specific dot-separated
-graphite metrics into labeled Prometheus metrics via a simple mapping language.
-This is read from a file specified by the `-graphite.mapping-config` flag. A
-mapping definition starts with a line matching the graphite metric in question,
-with `*`s acting as wildcards for each dot-separated metric component. The
-lines following the matching expression must contain one `label="value"` pair
-each, and at least define the metric name (label name `name`). The Prometheus
-metric is then constructed from these labels. `$n`-style references in the
-label value are replaced by the n-th wildcard match in the matching line,
-starting at 1. Multiple matching definitions are separated by one or more empty
-lines. The first mapping rule that matches a graphite metric wins.
+graphite metrics into labeled Prometheus metrics via YAML configuration file.  This file shares syntax and logic with [statsd_exporter](https://github.com/prometheus/statsd_exporter).  Please follow the statsd_exporter documentation for usage information.  However, graphite_exporter does not support *all* parsing features at this time.  Specific differences in functionality:
+
+- timer_type is not supported.  All metrics are assumed to be gauges as graphite does not distinguish.
+- quantiles are not supported
```

Comment:
isn't this implied by the first statement? what about histograms?

Reply:
Hmm I suppose that is all implied by just saying "only gauges" right now.  Good point.